Last-Update: 2015-11-03
Forwarded: not-needed
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Description: use "golang-github-fsouza-go-dockerclient-dev"

--- a/main.go
+++ b/main.go
@@ -14,9 +14,9 @@
 	"strconv"
 	"strings"
 	"time"
 
-	"github.com/docker/docker/opts"
+	"github.com/fsouza/go-dockerclient/external/github.com/docker/docker/opts"
 	flag "github.com/docker/docker/pkg/mflag"
 
 	dockerClient "github.com/fsouza/go-dockerclient"
 )
